lib/MIME/Types.rakumod

```diff
@@ -5,7 +5,7 @@ unit class MIME::Types;
 has %.exts;
 has %.types;
 
-method new (Stringy $mtfile) {
+method new (Str() $mtfile = %?RESOURCES<mime.types>) {
```

Comment:
%?RESOURCES stores paths (IO::Path), so it's probably better if you use that as the argument type. Also maybe split this into a multi... You don't need to check a IO::Path for existence, for instance, you might need it in the case of a random file name.

Reply:
Yeah I had seen `Stringy` before. seemed an unusual type. I am hoping to avoid breaking the API, I just tried `method new (IO::Path $mtfile = %?RESOURCES<mime.types>)` but when the constructor is passed a string, it breaks. How does one coerce `Str` to `IO::Path`?

Reply:
Oh, by multi you mean a multi sub? so effectively there are two constructors, one takes a `Str`, the other and `IO::Path`? sorry I am picking up the lingo as I go!

Reply:
That's correct.